### PR TITLE
ON UPDATE expressions

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/11441.
Related to https://github.com/cockroachdb/docs/issues/11028.

### CODE REVIEW

**Comment by ericharmeling** [comment]
@pawalt 

Thanks for taking the time to look over this! I've rewritten the "ON UPDATE expressions" section, following your feedback. I am a bit uncertain about the `ON UPDATE CASCADE` case - could you verify that what I've written is correct?

**Comment by pawalt** [comment]
@ericharmeling LGTM!

File: v21.2/add-column.md [hunk 1]
```diff
@@ -26,7 +26,37 @@ The user must have the `CREATE` [privilege](authorization.html#assign-privileges
...
+
+CockroachDB supports the following column qualifications:
+
+- [Column-level constraints](constraints.html)
+- [Collations](collate.html)
+- [Column family assignments](column-families.html)
+- [`DEFAULT` expressions](default-value.html)
+- <span class="version-tag">New in v21.2</span>: [`ON UPDATE` expressions](#on-update-expressions)
+
+### ON UPDATE expressions
```
Comment (stbof) [inline v21.2/add-column.md:41]:
```suggestion
### `ON UPDATE` expressions
```

**Review by stbof — APPROVED** [review]
See suggestion for formatting in heading.

File: v21.2/add-column.md (hunk 1 above)
Comment (ericharmeling) [reply v21.2/add-column.md]: Done

**Pull request merged**